test/plugin/test_out_google_cloud.rb

```diff
@@ -346,6 +346,14 @@ def test_statusz_endpoint
     end
   end
 
+  def test_monitoring_config
+    setup_gce_metadata_stubs
+    create_driver(%(
+      enable_monitoring  true
+      monitoring_type    not_prometheus
```

Comment:
For consistency with other tests, this config string constant should be pulled out into `Constants` and named.

Reply:
Forgot to respond to this one. It's not immediately obvious from the content of the config what is being tested. Giving the constant a meaningful name makes it easier to understand what it is about this config that makes it relevant to the test.

Reply:
I don't mind pulling it out into a constant. But I think it would be clearer if the input and expected result were next to each other, not in separate files.

Reply:
This is a good point. Having that constant in a separate file is consistent with other tests, but we could revisit this decision and possibly pull some constants into test local variables. Let's do that in a separate PR, though, and discuss the pros and cons.